src/packages/src/xroad/ubuntu/generic/xroad-proxy.config

```diff
@@ -2,7 +2,7 @@
 . /usr/share/debconf/confmodule
 
 get_prop() {
-  crudini --get "$1" '' "$2" 2>/dev/null || echo -n "$3"
+  awk -F "=" '/'"$2"'/ {print $2}' "$1" | tr -d ' '
```

Comment:
I think the following would be a bit more robust:
`awk -F'[\t ]*=[\t ]*' -v key="$2" '$1==key { sub(/[ \t]+$/, "", $2); print $2; exit }' "$1"`
This avoids removing spaces from the middle of the value (might be significant), takes into account that key could contain characters that are interpreted specially by the regex match, does not match a partial key, and stops on the first match.

Reply:
Yes, updated in https://github.com/nordic-institute/X-Road/pull/1024/commits/ea3f8808a4dbee35f307f88ff2d123c674edc76c